INSTALL.txt

```diff
@@ -80,8 +80,6 @@ Essential links about PrestaShop:
 * Official blog: https://www.prestashop.com/en/blog
 * Developer blog: https://build.prestashop.com/
 * Get community support: https://www.prestashop.com/forums/
```

Comment:
```suggestion
```

Reply:
Same for the user guide above: User Guide: https://doc.prestashop.com/display/PS17/User+Guide

Reply:
and the official blog

Reply:
User Guide stays in the project scope I believe.

I dont know if the forum belongs to the project or the company. Slack is for the project, it's a communication channel. Isn't the forum a similar communication channel?

Developer blog also belongs to the project.

My PR does not remove useless items, just addons related items 😉  .

Reply:
Everything that is not stored or managed by the open source project has to be removed

Reply:
> Everything that is not stored or managed by the open source project has to be removed

They told me to remove all links in the installer interface :sweat_smile: